search.html

```diff
@@ -0,0 +1,45 @@
+---
+layout: default
+title: Search JUGs
+---
+
+<script src="{{ '/assets/search.js' | relative_url }}"></script>
+
+<p>
+Search and browse the complete list of Java User Groups (JUGs) around the world.
+</p>
+
+<div style="margin-bottom: 20px;">
+    <input type="text" id="jug-search" placeholder="Search JUGs by name or country..." style="width: 100%; max-width: 400px; padding: 8px; font-size: 16px; border: 1px solid #ccc; border-radius: 4px;">
+</div>
+
+<div id="search-results" style="margin-bottom: 10px; font-style: italic; color: #666;"></div>
+
+{% assign jugs = site.jugs | sort: 'name' %}
+<ul id="jug-list">
+    {% for jug in jugs %}
```

Comment:
Don't you think we should hide all the JUGs by default and only show the results based on what the user types in? 🤔